panther_analysis_tool/main.py

```diff
@@ -1098,6 +1100,104 @@ def classify_analysis(
     return all_specs, invalid_specs
 
 
+def enrich_test_data(backend: BackendClient, args: argparse.Namespace) -> Tuple[int, str]:
```

Comment:
Can we put this in a new file in `panther_analysis_tool/command`? I would like to maintain the pattern of splitting up commands into their own files and putting them their instead of more stuff in main